daemons/lvmdbusd/utils.py

```diff
@@ -499,6 +499,28 @@ def validate_tag(interface, tag):
 			% (tag, _ALLOWABLE_TAG_CH))
 
 
+def add_no_notify(cmdline):
+	"""
+	Given a command line to execute we will see if `--config` is present, if it
+	is we will add the global/notify_dbus=0 to it, otherwise we will append it
+	to the end
+	:param cmdline:
```

Comment:
I think this should also specify ``:type cmdline:`` because otherwise it's not easy to determine it from the rest of the documentation here.